.travis.yml

```diff
@@ -4,7 +4,7 @@ node_js:
   - '8'
 
 install:
-  - npm i -g npm@5
+  - npm i -g npm@5.8
```

Comment:
You shouldn't need this. `npm@5` installs the latest 5 version, which is `5.10.0`.

Reply:
Well, if I set 5, `npm ci` fails in our pipelines, if I set it to 5.8, our pipelines works correct